### ISSUE TITLE
Heads up about new contributors

### ISSUE THREAD

**Issue by mbbroberg** [sensu/sensu-puppet#673]
**TL;DR** is the team at [LearnPuppet](http://learnpuppet.com/) started working through this repo's backlog as part of a plan for [Sensu Inc](https://blog.sensuapp.org/introducing-sensu-inc-9f1d2027c577?gi=c2112566ea09) to better support the Puppet community.
 
Hey folks :wave: 
(tagging recent committers @jaxxstorm, @devcfgc, @robbyt, @andyroyle)
 
I want to introduce myself - I joined Sensu this week to focus on our amazing Community. I also want to let you know what we’ve been up to recently and how we’re investing in the future of this awesome module and the sensu/puppet community. 
 
If you missed the news earlier this year, Heavy Water Operations (the company behind Sensu Core) [became Sensu, Inc](https://blog.sensuapp.org/introducing-sensu-inc-9f1d2027c577) and is now 100% focused on the Sensu product and community. This has been an exciting transition in general, but more specifically, it means we have a lot of new resources that we can dedicate to community projects like the sensu-puppet module.
 
While this module is already hugely successful for many of our users, there are a number of features that we’d love to see make it in. In order to jumpstart some of this work, @obfuscurity has engaged with @ghoneycutt and his team at LearnPuppet to knock out some of the more impactful issues ASAP, but more importantly, work with us in an ongoing effort to support and enhance this project.
 
So to summarize the new people: 
 
 * @obfuscurity is leading collaboration with Puppet, the company
 * @ghoneycutt will be regularly contributing to the project going forward
 * I’m (@mbbroberg) here for you as a Community lead
 
We want to support the community and continue to help it flourish. If there’s anything we can do to help, please let us know. You can reach me at matt `@sensu.io` or tag me on GitHub or [Twitter](https://twitter.com/mbbroberg). And feel free to close this issue once maintainers read it!
 
Thanks again for everything you do as part of our community. :heart:

**Comment by jaxxstorm** [comment]
Great to see extra hands here!
I seem to be the only one with access to the forge account at the moment, so let me know if I need to share with @ghoneycutt

**Comment by obfuscurity** [comment]
@jaxxstorm That would be awesome, can we also get access for @cwjohnston @mbbroberg and myself?

**Comment by jaxxstorm** [comment]
@obfuscurity yep. Is there a secure channel you'd prefer me to share via?

**Comment by obfuscurity** [comment]
@jaxxstorm Oh I assumed it would let you just connect via permissions. Let's start with @ghoneycutt for now and the others can request it directly as needed.

**Comment by ghoneycutt** [comment]
My public key is at http://pgp.mit.edu/pks/lookup?op=get&search=0x3DCF9E6861B486A8

**Comment by mbbroberg** [comment]
I'm so excited you're excited @jaxxstorm! I think this issue will be all set once @ghoneycutt has access, so feel free to close then. We'll talk more on Monday 👍

**Comment by Justin-DynamicD** [comment]
This is fantastic news ... look forward to seeing what comes from this.

**Comment by calebhailey** [comment]
@jaxxstorm any chance we can get that Puppet Forge access squared away today?

**Comment by jaxxstorm** [comment]
@calebhailey sorry for the delay, I'll get it sorted Monday.

@ghoneycutt is an email to the address in your profile okay?

**Comment by jaxxstorm** [comment]
I sent this over yesterday, let me know if there was any issue

**Comment by ghoneycutt** [comment]
Confirmed I have access to the Forge. Thank you!

**Comment by cwjohnston** [comment]
Thanks @jaxxstorm ! Looks like this has been handled so I'll go ahead and close this now.
